docs/user-guide/ze-usage.md

```diff
@@ -0,0 +1,310 @@
+# Using Zowe Explorer
+
+Use the content of this section to familiarize yourself with the extension, and make the best use of available options and features. The section contains usage tips, sample use cases for data sets, USS files, JOBs, and TSO commands.
+
+## Usage Tips
+
+Use the following tips to familiarize yourself with the extension and make the best use of it:
```

Comment:
Maybe could switch it up a bit to "Make the best use of the extension with the following tips:"